samples/browser/src/main/res/layout/activity_main.xml

```diff
@@ -13,7 +13,32 @@
         android:id="@+id/toolbar"
         android:layout_width="match_parent"
         android:layout_height="55dp"
-        android:background="#aaaaaa" />
+        android:background="#aaaaaa">
+
+        <mozilla.components.ui.autocomplete.InlineAutocompleteEditText
+            android:id="@+id/urlView"
+            android:layout_width="match_parent"
+            android:layout_height="40dp"
+            android:layout_weight="1"
+            android:layout_gravity="center_vertical"
+            android:imeOptions="actionGo|flagNoExtractUi|flagNoFullscreen"
+            android:background="@android:color/transparent"
+            android:inputType="textUri"
+            android:lines="1"
+            android:paddingTop="8dp"
+            android:paddingBottom="8dp"
+            android:paddingStart="8dp"
+            android:paddingEnd="8dp"
+            android:requiresFadingEdge="horizontal"
+            android:selectAllOnFocus="true"
+            android:textColor="#ffffff"
+            android:textColorHighlight="#ff9400"
+            android:textColorHint="#000000"
+            android:textSize="15sp"
+            android:importantForAutofill="no"
+            tools:targetApi="o" />
```

Comment:
As mentioned above. A consumer of the toolbar should just need to drop in the toolbar and not re-build the whole thing here. But that's not related to the PR; so we can just merge this for now.